### ISSUE TITLE
property based testing

### ISSUE THREAD

**Issue by asmodehn** [asmodehn/pyros#121]
https://github.com/HypothesisWorks/hypothesis-python

**Comment by fkromer** [comment]
You can probably use [hypothesis-ros](https://github.com/ros-testing/hypothesis-ros) (and the not yet existing [rospbt](https://github.com/ros-testing/rospbt)).

**Comment by asmodehn** [comment]
Yes I was thinking to use hypothesis-ros in the pyros-rosinterface package... There are things to simplify in testing there.
